src/main/java/apoc/load/LoadXls.java

```diff
@@ -246,7 +251,7 @@ private static Object getValue(Cell cell, CellType type) {
         switch (type) {
             case NUMERIC:
                 if (DateUtil.isCellDateFormatted(cell)) {
-                    return cell.getDateCellValue();
+                    return LocalDateTimeValue.localDateTime(cell.getDateCellValue().toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime());
```

Comment:
Torniamo semplicemente:
```suggestion
                    return cell.getDateCellValue();
```

Dopo di che nella classe `Mappings` andiamo a definire tutti i possibili Mapping con tutti i tipi date gestiti da Neo